cmd/build.go

```diff
@@ -119,6 +122,16 @@ func build(config *buildCommandConfig) error {
 	if execError != nil {
 		return execError
 	}
+	if config.push {
+		pushImage := buildImage
```

Comment:
Should `pushImage := config.tag` rather than `buildImage` here? Since we would need their username as part of the tag, for `docker push` to know where to push the image

Reply:
Isn't buildImage set to the value of tag above if one is present?

Reply:
But initially you set `buildImage := projectName`, and we only want to run `docker push` if `-t` has been specified and is not `config.tag != ""`, because doing `docker push` on `projectName` won't work